scipy/optimize/tests/test_linprog.py

```diff
@@ -2366,36 +2366,18 @@ def test_mip_rel_gap_passdown(self):
         assert not np.all(gap_diffs == 0)
 
     def test_semi_continuous(self):
-        # See issue #18106. This test if the integrality parameter
-        # is being checked correctly (status returns 0),
-        # when its > 1 it allows values to be 0 even if its out of bounds.
-
-        c = np.array([-1.])
-        A_ub = np.array([[1.]])
-        b_ub = np.array([0.5])
-        A_eq = np.zeros((0, 1), dtype=float)
-        b_eq = np.array([], dtype=float)
-        bounds = np.array([[1., 5.]])
-        integrality = np.array([2])
```

Comment:
I didn't think this first test added much, and I'd rather not add a separate test unless it checks something substantially different than the other. LMK if this was doing something important.